notebooks/text-to-image-genai/text-to-image-genai.ipynb

```diff
@@ -0,0 +1,393 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    def infer_lora(model_dir, prompt, adapters):
</pre><p>again for just changeing one argument you need to create new function, that is also not good from development point of view and UX as well as you recreate inference pipeline for each generation, it has time cost and everytime you spend additional time for first inference plugin initialization</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2472/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
fixed